regexes/device/mobiles.yml

```diff
@@ -3067,6 +3067,12 @@ Lenovo:
     - regex: 'Yoga Tablet([^;/]+) Build'
       model: 'Yoga Tablet$1'
       device: 'tablet'
+    - regex: 'Lenovo TB\-X704F Build/NMF26F'
+      model: 'Lenovo Tab 4 10 Plus'
+      device: 'tablet'
+    - regex: '(Lenovo TB[0-9]?\-[a-z0-9\-]+) Build'
```

Comment:
Please exclude the brand name from the match by using `(TB[0-9]?\-[a-z0-9\-]+) Build`. There's no need to match Lenovo at all, as this is already done with the base regex for the brand.

Reply:
+